docs/modules/trino/pages/usage_guide/catalogs/index.adoc

```diff
@@ -1,9 +1,11 @@
 = Catalogs
 
-Trino does not have a built-in catalog and instead `provides` connectors to external data sources. This allows Trino to connect to, read from and join a wide variety of data sources.
+Trino does not have a built-in catalog and instead `provides` connectors to external data sources.
```

Comment:
the `provides` should not ticket but maybe cursive?

Reply:
Removed the quotes entirely